com.unity.netcode.gameobjects/Editor/Simulator/NetworkEventsView.cs

```diff
@@ -15,12 +15,16 @@ public class NetworkEventsView : VisualElement
 
         INetworkEventsApi m_NetworkEventsApi;
         readonly NetworkSimulator m_NetworkSimulator;
+        const string k_LagSpikeValueString = "LagSpikeValue";
+        const string k_LagSpikeButtonStateString = "LagSpikeButtonState";
 
         public NetworkEventsView(NetworkSimulator networkSimulator)
         {
             m_NetworkSimulator = networkSimulator;
 
             AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(UXML).CloneTree(this);
+            LagSpikeDurationSlider.viewDataKey = k_LagSpikeValueString;
```

Comment:
Just a FYI to anyone else, this is temporary to fix the issue before we implement the UX "properly"

Reply:
Might suggest adding a comment to that effect but otherwise lgtm